tests/end2end/createEtForm.spec.js

```diff
@@ -74,14 +74,17 @@ describe('Create Event type form', function() {
 
         this.browser.login('#createtype')
         .selectByValue(strategyInput, 'hash')
+        .sleep()
```

Comment:
For how long does it sleep? Isn't it better to wait on a predictable condition instead of waiting on time?